houston/ardu/copter/commands.yml

```diff
@@ -2,6 +2,9 @@ commands:
   - name: MAV_CMD_NAV_WAYPOINT
     id: 16
     generator: circle_based_generator
+    next-allowed:
+      - "*"
+      - MAV_CMD_NAV_TAKEOFF
```

Comment:
Isn't this line redundant, or have I misunderstood the semantics of `*`?

Reply:
This basically means "all commands are allowed except `MAV_CMD_NAV_TAKEOFF`"
if * wasn't there it meant "nothing but `MAV_CMD_NAV_TAKEOFF`"